### ISSUE TITLE
Could not read log file of a drake server

### ISSUE THREAD

**Issue by jxsl13** [CytraL/twp#37]
<img width="914" alt="bildschirmfoto 2017-02-28 um 10 41 33" src="https://cloud.githubusercontent.com/assets/371944/23399993/a64c61aa-fda2-11e6-91fb-e7ed6225f425.png">

**Comment by CytraL** [comment]
Yes... this is because some mods change the debug output format... if only changes the date i can handle the exception and try get it with "YY-MM-DD HH:MM:SS" ... but if other mod uses other format this goes to break again...

I think that the solution is make an settings page for configure all this input stuff... but idk... i don't like to complicate everything because some mods changes the debug format... :/ and get extra support for ddnet... what happens if tomorrow appear other famous mod and changes the output format too? chaos! xD

But yes... this need a solution.
